packages/core/src/libraries/hook.ts

```diff
@@ -53,7 +55,10 @@ export const createHookLibrary = (queries: Queries) => {
 
     const hookEvent = eventToHook[event];
     const found = await findAllHooks();
-    const rows = found.filter(({ event }) => event === hookEvent);
+    const rows = found.filter(
+      ({ event, events, enabled }) =>
+        enabled && (events ? events.includes(hookEvent) : event === hookEvent) // For backward compatibility
```

Comment:
```suggestion
        enabled && (events?.includes(hookEvent) || event === hookEvent) // For backward compatibility
```

Reply:
Maybe not a good idea, the logic changed.

The original logic is:
```ts
if (!enabled) {
  return false;
}

if (events) {
  return events.includes(hookEvent);
}

return event === hookEvent;
```

The new version logic is:
```ts
if (!enabled) {
  return false;
}

if (events && events.includes(hookEvent)) {
  return true;
}

return event === hookEvent;
```

Reply:
good